src/main/java/seedu/duke/command/SearchModuleCommand.java

```diff
@@ -115,10 +116,12 @@ static boolean isOfferedInSemester(Module module, String toSearchSemester) {
      */
     public static List<Module> filterModuleSearch(String toSearchModuleCode, String toSearchLevel,
                                                   String toSearchSemester, String toSearchModuleTitle) {
+        assert toSearchModuleCode != null || toSearchModuleTitle != null : "At least either the module code or title "
```

Comment:
use exceptions for handling incorrect user input. Assertions should be for ensuring the program flow is correct

Reply:
exception used in constructor